src/agent.cc

```diff
@@ -95,13 +93,9 @@ Agent::~Agent() {
 
 std::string Agent::str() {
   std::stringstream ss;
-  ss << kind_ << "_" << prototype_
-     << " ( "
-     << "ID=" << id_
-     << ", implementation=" << spec_
-     << ",  name=" << prototype_
-     << ",  parentID=" << parent_id_
-     << " ) ";
+  ss << kind_ << "_" << prototype_ << " ( "
+     << "ID=" << id_ << ", implementation=" << spec_ << ",  name=" << prototype_
+     << ",  parentID=" << parent_id_ << " ) ";
```

Comment:
There are others like this:  In general, anywhere that we are using the stream operator for a label followed by a value, it makes most sense for those to be bound together.  I can't imagine that this can be automated.